apps/dashboard/components/app-users/smart-tables/app-users-table.tsx

```
@@ -32,170 +18,178 @@ const ITEMS_PER_PAGE = 100;
+      case 'tokens':
+        const totalTokens = personas.reduce((sum, persona) => {
+          if (persona.universalTokenList()) {
+            const portfolio = persona.universalTokenList();
+            const tokenCount = Object.values(portfolio).reduce(
+              (tokenCount: number, tokenList: any) => tokenCount + tokenList.length,
+              0
+            );
+            return sum + tokenCount;
+          }
+          return sum;
+        }, 0);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)